<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OhlcDlg</class>
 <widget class="QDialog" name="OhlcDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OhlcDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>schema:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>table:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>date column:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>open column:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>high column:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>low column:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>close column:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>volume column:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="schema_">
         <item>
          <property name="text">
           <string>ready</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>original</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="table_"/>
       </item>
       <item>
        <widget class="QLineEdit" name="date_col_">
         <property name="text">
          <string>date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="open_col_">
         <property name="text">
          <string>open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="high_col_">
         <property name="text">
          <string>high</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="low_col_">
         <property name="text">
          <string>low</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="close_col_">
         <property name="text">
          <string>close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="volume_col_">
         <property name="text">
          <string>vol</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OhlcDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OhlcDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
